docs/working-with-projects/project-configuration/configure-webhooks.md

```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
+There are two kinds of endpoints.  For `HTTP` the endpoint that receives the webhook should ideally have a webhook listener that is capable of interpreting the payload and acting upon the information it contains. For example, running a shell script.
+
+And for Slack endpoint, you should follow the [guide of Slack incoming webhook](https://api.slack.com/messaging/webhooks).
 
 ### Example Use Cases
 
 You can configure your continuous integration and development infrastructure so that it performs the following types of operations when it receives a webhook notification from Harbor.
 
-- Image push: 
+- Artifact push: 
   - Trigger a new build immediately following a push on selected repositories or tags.
-  - Notify services or applications that use the image that a new image is available and pull it.
-  - Scan the image using Clair.
-  - Replicate the image to remote registries.
+  - Notify services or applications that use the artifact that a new artifact is available and pull it.
+  - Scan the artifact using Clair.
+  - Replicate the artifact to remote registries.
 - Image scanning:
   - If a vulnerability is found, rescan the image or replicate it to another registry.
   - If the scan passes, deploy the image.
 
 ### Configure Webhooks
 
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+
 1. Go to **Projects**, select a project, and select **Webhooks**.
 
-    ![Webhooks option](../../../img/webhooks1.png)
+    ![Webhooks option](../../img/webhooks1.png)
+
+1. Select notify type `HTTP`, so the webhook will be send to a HTTP endpoint.
+
+1. Select events that you want to subscribe.
 
 1. Enter the URL for your webhook endpoint listener.
+
 1. If your webhook listener implements authentication, enter the authentication header. 
+
 1. To implement `HTTPS POST` instead of `HTTP POST`, select the **Verifiy Remote Certficate** check box.
 
-    ![Webhook URL](../../../img/webhooks2.png)
+    ![Webhook URL](../../img/webhooks2.png)
 
 1. Click **Test Endpoint** to make sure that Harbor can connect to the listener.
+
 1. Click **Continue** to create the webhook.
 
-When you have created the webhook, you see the status of the different notifications and the timestamp of the last time each notification was triggered. You can click **Disable** to disable notifications. 
+When you have created the webhook, you see the list of webhooks. If you click the arrow at the left of the row, can you see the status of the different notifications and the timestamp of the last time each notification was triggered. You can also manage the webhook by clicking the drop list button of `ACTION...` .
 
-{{< note >}}
-You can only disable and reenable all notifications. You cannot disable and enable selected notifications.
-{{< /note >}}
+You can modify the webhook, you can also `Enable` or `Disable` the webhook.
 
-![Webhook Status](../../../img/webhooks3.png)
+![Webhook Status](../../img/webhooks3.png)
```

Comment:
'../../../' is correct, this is a behavior of the system that hooks up to the website, basically the img is in its own directory, this needs to be changed back or image won't show properly on website